### PR TITLE
Add principles to website

### PR DESCRIPTION
**Pull request by azaroth42**
Adding principles to website, per @iherman's suggestion on the call today.

Review requested from people on the call to ensure it captures accurately, and reasonably succinctly, both the original proposals and the results of the discussion.

### CODE REVIEW

File: WorkMode/guiding_principles.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Follow existing standards and best practices, where possible and where they do not conflict with other principles.
+
+Between invention and reuse, pick reuse... unless that reuse would
+demonstrably harm adoption by being more complicated than necessary.
+
+## The underlying data model is RDF.
+
+If a feature comes up that can't be modeled with RDF as the underlying abstract data model, then we refer the feature to a future RDF Working Group for potential inclusion at that time. Similarly, we should ensure that the features of RDF are expressed in JSON-LD, to ensure that data can be round-tripped with confidence through different serializations.
+
```
Comment (gkellogg) [inline WorkMode/guiding_principles.md]: We need to define what we mean by “round-tripped”; does this include prefix selection, specific object embedding, use of encoding features, array ordering?

**Comment by workergnome** [comment]
Related to that—which way?  JSON-LD -> RDF -> JSON-LD, or RDF -> JSON-LD -> RDF?

I would think the latter is essential, but could be flexible on the former.

**Comment by gkellogg** [comment]
Certainly RDF -> JSON-LD -> RDF including compaction, but probably not framing. It’s the other direction that can be problematic.

**Comment by BigBlueHat** [comment]
@azaroth42 thinking we should mark these as "provisional" or "draft" until the group's made its way through the whole list. You OK with that?

**Comment by azaroth42** [comment]
Yep. How about "proposed" ?

**Comment by BigBlueHat** [comment]
I'll get this merged along with the draft minutes. Thanks @azaroth42!

**Comment by azaroth42** [comment]
Should wait on merge, at this stage, until after 7/13 call.

**Comment by azaroth42** [comment]
Updated from today's call, propose merge?

File: WorkMode/guiding_principles.md (hunk 1 above)
Comment (BigBlueHat) [reply WorkMode/guiding_principles.md]: @azaroth42 any chance we can get this clarification added?

**Pull request merged**